src/main/java/uk/gov/ons/validation/service/ProcessWranglerQuestionData.java

```diff
@@ -0,0 +1,146 @@
+package uk.gov.ons.validation.service;
+
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.lambda.AWSLambdaAsync;
+import com.amazonaws.services.lambda.AWSLambdaAsyncClient;
+import com.amazonaws.services.lambda.model.InvokeRequest;
+import com.amazonaws.services.lambda.model.InvokeResult;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import uk.gov.ons.validation.common.Constants;
+import uk.gov.ons.validation.entity.QuestionInputData;
+import uk.gov.ons.validation.entity.ValidationConfig;
+import lombok.extern.log4j.Log4j2;
+import uk.gov.ons.validation.entity.WranglerRequest;
+import uk.gov.ons.validation.entity.WranglerResponseData;
+import uk.gov.ons.validation.util.PropertiesUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.String.format;
+
+@Log4j2
+public class ProcessWranglerQuestionData {
+
+    private static final String SEND_MESSAGE = "Attempting to invoke %s with the json string %s.";
+    private static final String QUES_DER_CODE_MESSAGE = "Match Question Code %s and Derived Question code is %s.";
+    private static final String QUES_DER_VALUE_MESSAGE = "Match Question Code value %s and Derived Question code value is %s.";
+    private static final String MATCH_FOUND_QUEST = "Matching object found for Question and its code is %s.";
+    private static final String MATCH_FOUND_DERIVED_QUEST = "Matching object found for Derived Question and its code is %s.";
+
+
+    public void processQuestionAndDerivedData(WranglerRequest request) throws Exception {
+
+        log.info("Parsing Json Array to Java object: ");
+
+        //1. Building Validation Config this is separate feature. So hard coding it
```

Comment:
I would say that as this is just hardcoded then it should be extracted and put into a separate class so that it doesn't make this class look too busy.